Sources/SwifterSwift/CoreLocation/CLLocationExtensions.swift

```diff
@@ -63,6 +63,30 @@ public extension CLLocation {
 
         return (degrees + 360).truncatingRemainder(dividingBy: 360)
     }
+    
+    /// SwifterSwift: Check the distance between the start and end is less than or equal to the distance(=parameters).
+    ///
+    /// - Parameters:
+    ///   - from: Start location.
+    ///   - to: End location.
+    ///   - distance: Distance to check
+    ///   - unitLength: The unit of length. Default value is meters.
+    /// - Returns: True if the distance between the start and end is less than or equal to the distance(=parameters).
+    static func inRange(from start: CLLocation, to end: CLLocation, distance: Double, unitLength unit: UnitLength = .meters) -> Bool {
```

Comment:
Not sure if the static function is worth it, I would rather have only the method one.